Cauldron/Assets/Scripts/GameModes/PrestigeGameMode.cs

```diff
@@ -0,0 +1,40 @@
+using System;
+using CauldronCodebase.GameStates;
+using UnityEngine;
+using Zenject;
+
+namespace CauldronCodebase
+{
+    public class PrestigeGameMode : GameModeBase
+    {
+        [SerializeField] private int targetDays = 25;
+        protected string achievIdents;
```

Comment:
And this - protected abstract string achievId {get;}
And override it like: protected override string achievId => smth;